pages/lib/global_variables.py

```diff
@@ -1,6 +1,57 @@
 from dataclasses import dataclass
 from typing import Optional, List, Any
 from config import UnitSystem
+import colorcet as cc
+from plotly.colors import sequential as pseq
+
+# ---------- Shared palettes & conversion constants ----------
+BLUE_RED_YELLOW = ["#00b3ff", "#000082", "#ff0000", "#ffff00"]
```

Comment:
combine there into a class of (Enum) to keep the code more organised as we did for the other variables similarly to what we do below 
```
@dataclass
class VariableInfo:
    """Column metadata: default unit/range/color represent the common display;
    if SI or IP is provided, they will override the default values as needed."""

    col_name: str
    name: Optional[str] = None
```